tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+---
+meta:
+  title: Deploying multi-AZ Kubernetes cluster with Terraform and Kapsule
+  description: Learn how to deploy a multi-AZ Kubernetes cluster with Terraform and Kapsule
+content:
+  h1: Deploying multi-AZ Kubernetes cluster with Terraform and Kapsule
+  paragraph: Learn how to deploy a multi-AZ Kubernetes cluster with Terraform and Kapsule
+tags:  kubernetes multi-az terraform k8s
+categories:
+  - containers
+  - kubernetes
+dates:
+  validation: 2023-10-13
+  posted: 2023-10-13
+---
+
+Multi-AZ Kubernetes Kapsule is a configuration where a managed Kubernetes cluster is distributed across multiple isolated Availability Zones (AZs) within a Scaleway region, enhancing the resilience and availability of applications. 
+This configuration allows the system to withstand failures in one zone by replicating resources and workloads across multiple zones.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a domain managed on the Scaleway [Domains and DNS service](/network/domains-and-dns/quickstart/).
+  - You have installed [Terraform](/tutorials/terraform-quickstart/) on your local computer
+</Message>
+
+
+## Deploying the Kapsule Kubernetes cluster
+
+Start by creating a multi-AZ cluster on `fr-par` region, in a dedicated VPC and Private Network, with three pools of nodes, one in `fr-par-1`, one in `fr-par-2` and one in `fr-par-3`, with two nodes each.
+
+<Lightbox src="scaleway-k8s-kapsule-multi-az-1.webp" alt="" size="large" />
+
+1. Create a `main.tf` file that will contain the Terraform part of this project, and set the required providers and their versions
+    ```
+    terraform {
+        required_providers {
+            scaleway = {
+                source  = "scaleway/scaleway"
+                version = ">= 2.28.0"
+            }
+            kubernetes = {
+                source  = "hashicorp/kubernetes"
+                version = ">=2.23.0"
+            }
+            helm = {
+                source  = "hashicorp/helm"
+                version = "2.11.0"
+            }
+            local = {
+                source  = "hashicorp/local"
+                version = "2.4.0"
+            }
+            kubectl = {
+                source  = "gavinbunney/kubectl"
+                version = "1.14.0"
+            }
+        }
+        required_version = ">= 1.0.0"
+    }
+    ```
+
+2. Add the Scaleway provider and its configuration. Refer to the Scaleway Terraform provider documentation for more information on how to [configure the provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs).
+    ```
+    provider "scaleway" {
+        region  = "fr-par"
+    }
+    ```
+
+3. Now create a dedicated VPC and a Private Network where we will deploy the Kapsule cluster. VPC and Private Networks are region-wide, so you can use them to host a cluster spread on multiple availability zones.
```

Comment:
I'm not native speaker, but I have a feeling "to host a cluster spread on multiple availability zones" isn't right grammatically. 
Instead I'd have "to host a cluster _spreading_ on multiple availability zones."

Reply:
```suggestion
3. Now create a dedicated VPC and a Private Network where we will deploy the Kapsule cluster. VPC and Private Networks are region-wide, so you can use them to host a cluster spreading on multiple availability zones.
```

Reply:
```suggestion
3. Now create a dedicated VPC and a Private Network where we will deploy the Kapsule cluster. VPC and Private Networks are region-wide, so you can use them to host a cluster spreading across multiple availability zones.
```